testproject/testapp/validators.py

```diff
@@ -1,4 +1,7 @@
-def is_666(value):
-    from rest_framework import serializers
-    if value == '666':
-        raise serializers.ValidationError('Woops, 666 is not allowed.')
+from django.core.exceptions import ValidationError
+
+
+class Is665(object):
```

Comment:
I've pulled it from #176 so other teammate might get a credit for contribution (I'll probably split the PR before merge and blame her).